maker/maker.go

```diff
@@ -16,6 +16,12 @@ type Method struct {
 	Docs []string
 }
 
+type Parsed struct {
```

Comment:
I guess Parsed names is a little confusing. StructData sounds a bit better to me.